src/pke/lib/scheme/ckksrns/ckksrns-parametergeneration.cpp

```diff
@@ -82,27 +82,22 @@ bool ParameterGenerationCKKSRNS::ParamsGenCKKSRNSInternal(std::shared_ptr<Crypto
         //                  e.g.1, assertion: prime size < registerSize (we may need at least 1-2 bit gap)
         //                  e.g.2, prime size > ??? if numPrimes > ???
         if (compositeDegree > 2 && scalingModSize < 60) {
-            std::string errorMsg = "COMPOSITESCALING Warning:";
-            errorMsg +=
-                "There will not be enough prime moduli for composite degree > 2 and scaling factor < 60 -- prime moduli too small.";
-            errorMsg +=
-                "Prime moduli size must generally be greater than 19, especially for larger multiplicative depth.";
-            errorMsg += "Consider increasing the scaling factor (scalingModSize) or the register word size.";
-            errorMsg += "Also, feel free to use COMPOSITESCALINGMANUAL at your own risk.";
+            std::string errorMsg = "Prime moduli is too small. Its size must generally be greater than 19,";
```

Comment:
Prime moduli are too small - moduli is plural. Please change to plural everywhere here.